### PR TITLE
documentation for juxt, complement, fNil and trampoline

### PR DESCRIPTION
**Pull request by ashishsantikari**

### CODE REVIEW

File: ljsp/docs/complement.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+id: complement
+title: complement
+---
+
+Takes a fn f and returns a fn that takes the same arguments as f, has the same effects, if any, and returns the opposite
```
Comment (joe-finreach) [inline ljsp/docs/complement.md:6]: Another mistake in the docs, as with `fNil` below, let's make sure we mark variables as inline code, e.g.:

> Takes a fn `f` and returns a fn that takes the same arguments as `f`...

**Review by joe-finreach — CHANGES_REQUESTED** [review]
Right on!!!

File: ljsp/docs/complement.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+const notEq = complement(_.eq);
+
+console.log(notEq("a", "b"));
+// true
+```
+
+#### Parameters
+
+- `args` | `Function` Variadic. One or more functions.
```
[the remark this answers is not in the document]
Comment (ashishsantikari) [reply ljsp/docs/complement.md]: fixed

File: ljsp/docs/complement.md (hunk 1 above)
Comment (ashishsantikari) [reply ljsp/docs/complement.md]: fixed

**Pull request merged**